common/metrics/defs.go

```diff
@@ -3304,6 +3322,16 @@ var PersistenceLatencyBuckets = tally.DurationBuckets([]time.Duration{
 	60 * time.Second,
 })
 
+// GlobalRatelimiterUsageHistogram contains buckets for tracking how many ratelimiters are
+// in which state (startup, healthy, failing).
+var GlobalRatelimiterUsageHistogram = tally.ValueBuckets{
+	1, 2, 5, 10,
+	25, 50, 100,
+	250, 500, 1000,
+	1250, 1500, 2000,
+	// TODO: almost certainly want more, but how many?
+}
```

Comment:
4 collections * 500 domains = 2000, so if 500+ domains are using all API categories every 3 seconds we'd see the 2000+ bucket used.

We _do_ have some clusters this large, but I don't believe they're used heavily enough to exceed this, much less to a degree that we would care.  But this is trivial to adjust if there are any other thoughts.